BuildLoader.py

```diff
@@ -1497,6 +1497,56 @@ def cmd_clean(args):
             files.extend ([
             ])
 
+        def GetCopyList (driver_inf):
+            fd = open (driver_inf, 'r')
+            lines = fd.readlines()
+            fd.close ()
+
+            have_copylist_section = False
+            copy_list      = []
+            for line in lines:
+                line = line.strip ()
+                if line.startswith('['):
+                    if line.startswith('[UserExtensions.SBL."CopyList"]'):
+                        have_copylist_section = True
+                    else:
+                        have_copylist_section = False
+
+                if have_copylist_section:
+                    match = re.match("^(.+)\s*:\s*(.+)", line)
+                    if match:
+                        copy_list.append((match.group(1).strip(), match.group(2).strip()))
+
+            return copy_list
+
+        if args.board:
+            for index, name in enumerate(board_names):
+                if args.board == name:
+
+                    board  = module_names[index].Board()
+                    if hasattr(board, 'FSP_INF_FILE'):
+                        fsp_inf = board.FSP_INF_FILE
+                    else:
+                        fsp_inf = 'Silicon/%s/FspBin/FspBin.inf' % board.SILICON_PKG_NAME
+
+                    if os.path.exists(fsp_inf):
```

Comment:
If the workspace is not under SblOpen, fsp_inf might not be found since it is a relative path to current workspace.
you might check os.path.join (sbl_dir, fsp_inf) and os.path.join (os.environ['PLT_SOURCE'], fsp_inf) if 'PLT_SOURCE' in os.environ. Same for other INF file

Reply:
@gdong1, update, pls review. thanks